tensorboard/plugins/mesh/summary.py

```diff
@@ -79,6 +84,16 @@ def _get_json_config(config_dict):
   return json_config
 
 
+def _get_components_bitmap(tensors):
+  """Creates bitmap for all existing components of the summary."""
+  components = 0
+  for tensor, content_type in tensors:
+    if tensor is None:
+      continue
+    components = components | (1 << content_type)
```

Comment:
Is `ContentType.UNDEFINED` a valid member of the mask? If so, what does
it represent/when might it be in the mask/what would the plugin do in
response? If not, any consideration for using `1 << (content_type - 1)`
instead, in part to make it obvious that `UNDEFINED` is not maskable?
(Just a suggestion; if you’ve considered this and disregarded it, no
problem, but I’d be interested to hear why.)

Reply:
I'd just remove it in followup PR since it is not being used